ovirt-engine-dwh.spec.in

```diff
@@ -154,8 +154,7 @@ Summary:	%{product_name} Grafana integration setup
 Group:		Virtualization/Management
 Requires:	ovirt-engine-setup-plugin-ovirt-engine-common >= 4.5.3
 Requires:	%{name}-setup = %{version}-%{release}
-Requires:	grafana >= 7.3
-Requires:	grafana-postgres >= 7.3
+Requires:	grafana >= 9.2.10-15
```

Comment:
There might be a problem with such requirement, because grafana 9.2.10-15 is available only in CentOS Stream, but not in any previous stable releases (RHEL, Rocky or Alma < 8.10 or 9.4).

Grafana selinux support was introduced in 9.2.10-10 release (RHEL/Rocky/Alma 8.9 or 9.3), so all previous Grafana version >= 7.3 should work fine. So the question is if we should not leave version dependency in spec file as it work before and rather add logic to engine-setup part:

1. Detect available grafana version using DNF query
2. If Grafana version is >= 7.3 and < 9.2.10-10, proceed as before
3. If Grafana version is >= 9.2.10-10 and < 9.2.10-15, fail the setup with error to upgrade to 9.2.10-15+
4. If Grafana version is >= 9.2.10-15 the enable selinux boolen `grafana_can_tcp_connect_postgresql_port`

Reply:
Ok, i will try to prepare this steps.

Reply:
@mwperina 
I updated solution to have grafana version check, but i not sure that correctly raise error to ask user for grafana update. Look please.

Reply:
I don't remember details anymore, sorry. If we do not versionlock grafana, user can update it without engine-setup.